subprojects/packagefiles/cmark-gfm/meson.build

```diff
@@ -0,0 +1,121 @@
+project(
+    'cmark-gfm',
+    'c',
+    version: '0.29.0.gfm.6',
```

Comment:
Can you add `meson_version: '>=0.49.0',`? That's the minimum required version of Meson based on the features this uses -- always better to be explicit IMO.

Reply:
Done.